scripts/build_pyodide_wheels.py

```diff
@@ -26,8 +31,24 @@
     default=False,
     help="Don't install package dependencies.",
 )
+parser.add_argument(
+    "--verify-clean",
+    action="store_true",
+    default=False,
+    help="Check if panel folder is clean before running.",
+)
 args = parser.parse_args()
 
+if args.verify_clean:
```

Comment:
The test is not clean, so I have added it as an option and enforced it on the build.